src/protocols/rdp/settings.c

```diff
@@ -371,6 +372,13 @@ enum RDP_ARGS_IDX {
      */
     IDX_DISABLE_GLYPH_CACHING,
 
+    /**
+     * "true" if the RDP Graphics Pipeline Extension should not be used, and
+     * traditional RDP graphics should be used instead, "false" or blank the
```

Comment:
I think some minor rewording would make this comment easier to read - maybe just adding "if" between "blank" and "the".

Reply:
Yep, I definitely a word. I'll fix.

Reply:
Fixed via rebase.